server/src/main/java/org/apache/uniffle/server/storage/HadoopStorageManager.java

```diff
@@ -57,12 +57,14 @@ public class HadoopStorageManager extends SingleStorageManager {
   private static final Logger LOG = LoggerFactory.getLogger(HadoopStorageManager.class);
 
   private final Configuration hadoopConf;
+  private final String shuffleServerId;
   private Map<String, HadoopStorage> appIdToStorages = JavaUtils.newConcurrentMap();
   private Map<String, HadoopStorage> pathToStorages = JavaUtils.newConcurrentMap();
 
   HadoopStorageManager(ShuffleServerConf conf) {
     super(conf);
     hadoopConf = conf.getHadoopConf();
+    shuffleServerId = conf.getString(ShuffleServerConf.SHUFFLE_SERVER_ID, "shuffleServerId");
```

Comment:
Is the default value right? Why not fast fail?

Reply:
This is also the case when flushing. I think it’s better to keep both sides consistent. @zuston 
https://github.com/apache/incubator-uniffle/blob/ae72831615b05b56055dcfda8f129189792130ba/server/src/main/java/org/apache/uniffle/server/ShuffleFlushManager.java#L202-L204

Reply:
Got it.